appveyor.yml

```diff
@@ -0,0 +1,27 @@
+build: false
+shallow_clone: false
+platform: 'x86'
+clone_folder: c:\projects\symfony
+environment:
+  global:
+    PHP: "C:/PHP"
+init:
+  - SET PATH=C:\php\;%PATH%
+install:
+  - cd c:\
+  - ps: Start-FileDownload 'http://ci.cakephp.org/php.zip'
```

Comment:
Why do we download from CakePHP and not http://windows.php.net/downloads/releases/php-5.6.12-nts-Win32-VC11-x86.zip ?